scripts/validate-configurator-output/app.py

```diff
@@ -0,0 +1,35 @@
+from dataclasses import dataclass
+import json
+
+import cfn_flip
+
+filename = "test.json"
+with open(filename) as f:
+    stuff = json.load(f)
+
+def get_dict_from_params_list(params_list):
+    params_keys = []
+    for param in params_list:
+        key, value = param.split('=')
+        params_keys.append(key)
+    return params_keys
```

Comment:
I would think this should return a `dict` if named `get_dict_...`; let's change method name or at least add a return TypeHint if returning a list; e.g.
```python
def get_dict_from_params_list(params_list) -> list[str]:
```